<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="log">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>로그가 여기에 표시됩니다</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="count_5">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>70</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>5회</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLCDNumber" name="score_lcd">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
   <property name="value" stdset="0">
    <double>12345.000000000000000</double>
   </property>
  </widget>
  <widget class="QRadioButton" name="count_10">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>90</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>10회</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </widget>
  <widget class="QLabel" name="result">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>401</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>결과 : 미실행 됨</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start_btn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>시작하기</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="count_20">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>110</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>20회</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="log_view">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>로그 뷰어 여부</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_btn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>탈출!</string>
   </property>
  </widget>
  <widget class="QLabel" name="count_num">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>시도 횟수 설정</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
